app/dashboard/models.py

```diff
@@ -1299,6 +1305,43 @@ def total_reserved_length_label(self):
             return ''
 
 
+@receiver(post_save, sender=Bounty, dispatch_uid="post_bounty")
+def post_save_bounty(sender, instance, created, **kwargs):
+    final_state = instance.bounty_state in ['done', 'cancelled']
+    if not final_state and instance.hypercharge_mode and instance.metadata.get('hyper_tweet_counter', False) is False:
+        instance.metadata['hyper_tweet_counter'] = 0
+
+        title = f'Work on "{instance.title}" and receive {floatformat(instance.value_true)} {instance.token_name}'
+
+        # Feature bounty on hackathon/prize explorer
+        instance.is_featured = True
+        instance.featuring_date = timezone.now()
+        instance.hyper_next_publication = timezone.now()
+
+        # Publish and pin on townsaquare
+        profile = Profile.objects.filter(handle='owocki').first()
```

Comment:
ill probably wanna try/except this. not everyone has an  `owocki` on their local